M/Mosek/Versions.toml

```diff
@@ -31,6 +31,12 @@ git-tree-sha1 = "eae97e18c971edfcb68af3e0c8c370af9ecc426f"
 ["0.9.11"]
 git-tree-sha1 = "032bd53d5d2c1a28ca4b684d0bcb43bf50ed1d5f"
 
+["0.9.12"]
+git-tree-sha1 = "e88c1d374ae25842d26debb991cfda7eec9c76be"
+
+["0.9.13"]
+git-tree-sha1 = "5f6244ebded09e55cda59969d07f4f74f8fc9eb1"
```

Comment:
These look like the commits git hashes, not the tree hashes

Reply:
Indeed, thanks! I have corrected it in https://github.com/JuliaRegistries/General/pull/52054